### ISSUE TITLE
Schedule modal missing confirmation button

### ISSUE THREAD

**Issue by leeuwd** [sanity-io/sanity-plugin-scheduled-publishing#96]
**Describe the bug**

Since the Studio UI upgrade (https://www.sanity.io/changelog/f75445eb-3c3e-4d1e-986d-71a61223aa83), the scheduling modal lacks a confirmation/save button.

**Screenshots**

<img width="700" alt="image" src="https://github.com/sanity-io/sanity-plugin-scheduled-publishing/assets/1755190/8ef20307-52ee-486a-a110-9c0261c98d03">

**Which versions of Sanity are you using?**

```
├─ sanity-diff-patch@3.0.2
├─ sanity-plugin-dashboard-widget-vercel@2.0.1
├─ sanity-plugin-google-translate@3.0.0
├─ sanity-plugin-hotspot-array@1.0.1
├─ sanity-plugin-iframe-pane@2.6.1
├─ sanity-plugin-media@2.2.4
├─ sanity-plugin-utils@1.6.2
├─ sanity@3.23.1
```

`@sanity/cli (global)  3.21.3 (latest: 3.23.1)`

**What operating system are you using?**

**Which versions of Node.js / npm are you running?**

```
9.6.7
v18.17.0
```

**Previous versions**

![image](https://github.com/sanity-io/sanity-plugin-scheduled-publishing/assets/1755190/f93687fc-a1f2-4864-aaf0-5251d6b522f2)

**Comment by SachinthaI** [comment]
The same issue exists in my studio as well.

**Comment by tavaneftekhar** [comment]
Same issue here as well, with the same updated version of the Studio.

**Comment by rohit-jaxxon** [comment]
Same issue here as well

**Comment by aqilusman45** [comment]
I am also facing the same issue with the below packages
    "@sanity/client": "6.4.7",
    "@sanity/demo": "1.0.2",
    "@sanity/ui": "^1.8.2",
    "@sanity/vision": "3.15.1",
     "next-sanity": "^5.5.4",
      "sanity": "^3.16.7",

**Comment by darrenjacoby** [comment]
Same issue

**Comment by webprismdevin** [comment]
Downgrading to v3.22.0 (old UI) brought the schedule button back.

Be forewarned - I can't deploy my Studio now

**Comment by leeuwd** [comment]
Tagging you here @robinpyon since you seem to be the main contributor. Is this plugin still alive?

**Comment by robinpyon** [comment]
Thank you all for reporting, sincere apologies for the latency. 

A fix for this issue has landed and will be available in next week's Studio release. Appreciate your patience here!

**Comment by robinpyon** [comment]
This has been fixed as of Studio `3.24.1`! Please shout if you're still seeing similar issues.
